packages/internal/src/__tests__/graphqlCodeGen.test.ts

```diff
@@ -0,0 +1,218 @@
+import fs from 'fs'
+import path from 'path'
+
+import {
+  generateTypeDefGraphQLApi,
+  generateTypeDefGraphQLWeb,
+} from '../generate/graphqlCodeGen'
+import { generateGraphQLSchema } from '../generate/graphqlSchema'
+
+const FIXTURE_PATH = path.resolve(
+  __dirname,
+  '../../../../__fixtures__/example-todo-main'
+)
+
+beforeAll(() => {
+  process.env.RWJS_CWD = FIXTURE_PATH
+})
+
+afterAll(() => {
+  delete process.env.RWJS_CWD
+})
+
+test('Generate gql typedefs web', async () => {
+  await generateGraphQLSchema()
+
+  const webPaths = await generateTypeDefGraphQLWeb()
+  const gqlTypesWebOutput = fs.readFileSync(webPaths[0], 'utf-8')
```

Comment:
Minor suggestion. what do you think of mocking fs.writeFileSync here instead of using the index here?

Reply:
I really like this idea. Saves on disk I/O for the test case which is always a good idea.